elasticsearch.md

```
@@ -799,11 +799,86 @@ So that Kibana will forward the necessary headers to Elasticsearch.
+    not_any_of: ["group1", "group2"]
+```
+
+With deprecated syntax (without being wrapped inside the `user_belongs_to_groups` section):
+- `groups_not_any_of: ["group1", "group2"]`
+
+##### `not_all_of`
+
+The ACL block will match when the user does not belong to all the specified groups.
+
+Looking at the example below (and notice the difference with `not_any_of` rule):
+- ACL block will match for user that belongs to `group0`
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)